specification/cluster/stats/types.ts

```diff
@@ -281,4 +283,27 @@ export class OperatingSystemMemoryInfo {
   total_in_bytes: long
   used_in_bytes: long
   used_percent: integer
+  /** @since 7.16.0 */
+  adjusted_total_in_bytes?: long
+}
+
+export class ClusterIndexingPressure {
```

Comment:
If possible, use shorter names, for example:
```suggestion
export class IndexingPressure {
```